zerver/lib/import_realm.py

```diff
@@ -385,7 +388,8 @@ def import_uploads(import_dir: Path, processing_avatars: bool=False,
             bucket_name = settings.S3_AVATAR_BUCKET
         else:
             bucket_name = settings.S3_AUTH_UPLOADS_BUCKET
-        import_uploads_s3(bucket_name, import_dir, processing_avatars=processing_avatars)
+        import_uploads_s3(bucket_name, import_dir, processing_avatars=processing_avatars,
+                          processing_emojis=processing_emojis)
```

Comment:
Are these all separate bugs? I would at least itemize them in the commit message, and maybe even separate them into individual commits. Do the tests from previous commits pass before applying this commit? If not, then I would reorder this (or these individually if separating them) so the code is fixed first, and then the tests are applied after.

Reply:
All these are minor bugs. Basically `import_uploads_s3` and `import_uploads_local` should follow the same pattern for proper functioning, and I noticed that `import_uploads_s3` wasn't functioning properly before, so I fixed these bugs.

Regarding the tests, Previously this didn't have a coverage, but I am adding a test in a later commit (18bb81f6de1b5175f9a33a6a2fcfb37bc6fecc90 ) for this, which gives this coverage.